py_proc.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import hashlib
+import logging
+
+from sawtooth_sdk.processor.handler import TransactionHandler
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.processor.exceptions import InternalError
+from sawtooth_sdk.processor.core import TransactionProcessor
+
+LOGGER = logging.getLogger(__name__)
+
+FAMILY_NAME = "simplewallet"
+
+
+def _hash(data):
+    return hashlib.sha512(data).hexdigest()
+
+
+sw_namespace = _hash(FAMILY_NAME.encode('utf-8'))[0:6]
+
+
+class SWTransactionHandler(TransactionHandler):
```

Comment:
Lets use the "SimpleWalletTransactionHandler" here

Reply:
Done